Document PAT restriction on service token creation (#9636)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## July 2026

> - **Behavior change:** You can no longer create a [service token](/docs/dbt-apis/service-tokens) using an account-scoped [personal access token](/docs/dbt-apis/user-tokens) (PAT). Requests to the service tokens endpoint that authenticate with a PAT now return a `400` error. Use a service token to create other service tokens instead.

Reviewer: ```suggestion
- **Behavior change:** You can no longer create a [service token](/docs/dbt-apis/service-tokens) using an account-scoped [personal access token](/docs/dbt-apis/user-tokens) (PAT). Requests to the service tokens endpoint authenticated with a PAT now return a `400` error. Use a service token to create new ones instead.
```
